test/unit/org/apache/cassandra/tools/nodetool/stats/NodetoolTableStatsTest.java

```diff
@@ -78,11 +78,14 @@ public void testMaybeChangeDocs()
                         "                [(-pp | --print-port)] [(-pw <password> | --password <password>)]\n" + 
                         "                [(-pwf <passwordFilePath> | --password-file <passwordFilePath>)]\n" + 
                         "                [(-u <username> | --username <username>)] tablestats\n" + 
-                        "                [(-F <format> | --format <format>)] [(-H | --human-readable)] [-i]\n" + 
-                        "                [(-s <sort_key> | --sort <sort_key>)] [(-t <top> | --top <top>)] [--]\n" + 
-                        "                [<keyspace.table>...]\n" + 
+                        "                [(-B | --boundary-check)] [(-F <format> | --format <format>)]\n" +
+                        "                [(-H | --human-readable)] [-i] [(-s <sort_key> | --sort <sort_key>)]\n" +
+                        "                [(-t <top> | --top <top>)] [--] [<keyspace.table>...]\n" +
                         "\n" + 
-                        "OPTIONS\n" + 
+                        "OPTIONS\n" +
+                        "        -B, --boundary-check\n" +
```

Comment:
Is there a rule when should we use capital letter and when not? I see both and if there is no rule I would leave it lower-case as more intuitive, maybe?

Reply:
Not sure if there is a rule. The nodetool options naming does not seem to be very consistent among all the commands. 
I do not have preferences. Lower-case sounds good to me. It is a more common pattern for general cli.